pkg/installer/installer.go

```diff
@@ -24,39 +24,42 @@ import (
-	SkipEtcdEndpointsValFlag       = "skip-etcd-endpoints-validation"
-	SkipStosClusterFlag            = "skip-stos-cluster"
-	EtcdTLSEnabledFlag             = "etcd-tls-enabled"
-	EtcdSecretNameFlag             = "etcd-secret-name"
-	StosConfigPathFlag             = "stos-config-path"
-	EtcdNamespaceFlag              = "etcd-namespace"
-	StosOperatorNSFlag             = "stos-operator-namespace"
-	StosClusterNSFlag              = "stos-cluster-namespace"
-	EtcdStorageClassFlag           = "etcd-storage-class"
-	AdminUsernameFlag              = "admin-username"
-	AdminPasswordFlag              = "admin-password"
-	PortalClientIDFlag             = "portal-client-id"
-	PortalSecretFlag               = "portal-secret"
-	PortalTenantIDFlag             = "portal-tenant-id"
-	PortalAPIURLFlag               = "portal-api-url"
-	EnablePortalManagerFlag        = "enable-portal-manager"
+	StackTraceFlag                    = "stack-trace"
+	SkipNamespaceDeletionFlag         = "skip-namespace-deletion"
+	SkipExistingWorkloadCheckFlag     = "skip-existing-workload-check"
+	StosVersionFlag                   = "stos-version"
+	EtcdOperatorVersionFlag           = "etcd-operator-version"
+	K8sVersionFlag                    = "k8s-version"
+	WaitFlag                          = "wait"
+	DryRunFlag                        = "dry-run"
+	StosOperatorYamlFlag              = "stos-operator-yaml"
+	StosClusterYamlFlag               = "stos-cluster-yaml"
+	StosPortalConfigYamlFlag          = "stos-portal-config-yaml"
+	StosPortalClientSecretYamlFlag    = "stos-portal-client-secret-yaml"
+	EtcdOperatorYamlFlag              = "etcd-operator-yaml"
+	EtcdClusterYamlFlag               = "etcd-cluster-yaml"
+	ResourceQuotaYamlFlag             = "resource-quota-yaml"
+	IncludeEtcdFlag                   = "include-etcd"
+	EtcdEndpointsFlag                 = "etcd-endpoints"
+	SkipEtcdEndpointsValFlag          = "skip-etcd-endpoints-validation"
+	SkipStosClusterFlag               = "skip-stos-cluster"
+	EtcdTLSEnabledFlag                = "etcd-tls-enabled"
+	EtcdSecretNameFlag                = "etcd-secret-name"
+	StosConfigPathFlag                = "stos-config-path"
+	EtcdNamespaceFlag                 = "etcd-namespace"
+	StosOperatorNSFlag                = "stos-operator-namespace"
+	StosClusterNSFlag                 = "stos-cluster-namespace"
+	EtcdStorageClassFlag              = "etcd-storage-class"
+	AdminUsernameFlag                 = "admin-username"
+	AdminPasswordFlag                 = "admin-password"
+	PortalClientIDFlag                = "portal-client-id"
+	PortalSecretFlag                  = "portal-secret"
+	PortalTenantIDFlag                = "portal-tenant-id"
+	PortalAPIURLFlag                  = "portal-api-url"
+	EnablePortalManagerFlag           = "enable-portal-manager"
+	InstallLocalPathProvisionerFlag   = "install-local-path-storage-class"
+	UninstallLocalPathProvisionerFlag = "uninstall-local-path-storage-class"
```

Comment:
Same as earlier, maybe a single `include-local-path-storage-class` flag?

Reply:
In terms of making things clear, I'd argue that the subcommand `install` or `uninstall` before the flag _should_ already be pretty clear. But I certainly take the point.

As for the actual naming, I'd like to keep the config options and flags the same, so if we have `includeLocal....` in the config then the flag should be `--include-local...` too. 
So if we want separate flags, we should also have separate config fields.... but then this goes against the precedent set by `includeEtcd`/`--include-etcd` 😄 

Either way is fine by me really, I just want to be consistent... Others can chime in if they have a strong preference